### DISCUSSION TITLE
Display fehlerhaft - Hardwaredefekt oder Softwareproblem?

### DISCUSSION THREAD

**Discussion by NintendoTito** [tbnobody/OpenDTU#1131] [Q&A]
Hallo zusammen. Gerade mal bei nem Kollegen ne OpenDTU geflasht. Und zwar nach der "neuen" Variante, die Bin Datei mit der ESP32 im Namen. Und als Device Profile die nodemcu_esp32.json genutzt. Und unter Einstellungen dann die vermeintlich richtige Pin Konfiguration genutzt. Funzt auch soweit alles. Das Display ist aber leider an einer Seite verzerrt (Immer die gelbe Seite) Display kaputt? Oder doch Einstellungssache? 🤔

In dem gelben Bereich wird die Schrift sozusagen "Interlaced" dargestellt, jede zweite Pixelreihe scheint schwarz zu sein. Die Kontakte und Verbindungen wurden bereits überprüft. 

![PXL_20230708_131859930](https://github.com/tbnobody/OpenDTU/assets/128979769/96472eb8-eaa3-4785-8175-6b78853afb99)
![PXL_20230708_131846216 MP](https://github.com/tbnobody/OpenDTU/assets/128979769/28a80be7-3ee2-40f4-8385-84acd1166f8d)

**Comment by tbnobody** [comment]
Was für ein Controller wird denn da im Displaytyp verwendet? Wurde entsprechend der korrekte Typ im Device Profile ausgewählt?

**Reply by NintendoTito** [reply]
![Screenshot_20230709-114133](https://github.com/tbnobody/OpenDTU/assets/128979769/93f4c454-7bea-40e5-8fde-8f3326f0100e)

Es ist dieses Display:
https://m.de.aliexpress.com/item/1005004860363043

Die Nodemcu_esp32.json wurde hochgeladen. Unter Device Profile weiß ich gerade nicht genau, habe die DTU gerade nicht hier. War aber die einzige die sinnvoll erschien, also im Namen war NRF, also das Funkmodul und SSD1306 im Namen.

**Reply by NintendoTito** [reply]
Der Screenshot ist von meiner DTU. Der oben sichtbare Profilname war bei meinem Kollegen so nicht vorhanden.
